lib/request.js

```diff
@@ -494,7 +502,12 @@ exports = module.exports = internals.Request = class {
         }
 
         this.info.completed = Date.now();
+
         this._core.events.emit('response', this);
+
+        if (this._route._extensions.onPostResponse.nodes) {
+            this._invoke(this._route._extensions.onPostResponse, { ignoreResponse: true });
+        }
```

Comment:
Why the conditional? It should work the same without.

Reply:
Should be faster avoiding the call + promise generation.